src/schema/TypeORMConverter.ts

```diff
@@ -115,6 +118,11 @@ export function entityToCreateInput(entity: EntityMetadata): string {
           @TypeGraphQLField(type => ${graphQLDataType}, ${nullable})
           ${column.propertyName}${tsRequired}: ${graphQLDataType};
        `;
+      } else if (graphQLDataType === 'JSONObject') {
+        fieldTemplates += `
+          @TypeGraphQLField(type => GraphQLJSONObject, ${nullable})
+          ${column.propertyName}${tsRequired}: any;
+       `;
```

Comment:
Could we just OR this above?  Something like:

```
if (column.enum || column.isJSON) {
```

It would likely require that we teach `columnToTypeScriptType` to return the appropriate types for both enum and JSON, but it would be cleaner.

Reply:
Changed any to JSON

Reply:
I think the problem here is that the TypeGraphQLField(type =>) and the tsType are the same for enums and different for GraphQLJSON objects.

if (column.enum) {
        fieldTemplates += `
          @TypeGraphQLField(type => ${graphQLDataType}, ${nullable})
          ${column.propertyName}${tsRequired}: ${graphQLDataType};
       `;
      } else if (graphQLDataType === 'JSON') {
        fieldTemplates += `
          @TypeGraphQLField(type => GraphQLJSONObject, ${nullable})
          ${column.propertyName}${tsRequired}: JSON;
       `;

Reply:
Ok, I updated columnToTypeScriptType to return JSON correctly. But I sti

Reply:
could you take another look at this file. I think it's cleaner